package.json

```diff
@@ -129,7 +130,6 @@
     "querystring": "^0.2.0",
     "rc-time-picker": "^2.4.1",
     "react": "^16.8.6",
-    "react-appinsights": "^1.0.2",
```

Comment:
this isn't used in codebase.  Deprecated too.  Maybe worth investigating https://www.npmjs.com/package/@microsoft/applicationinsights-react-js at a later date...